src/analysis/inner_product_space/area_form.lean

```diff
@@ -0,0 +1,440 @@
+/-
```

Comment:
I am not sure about the name of the file, since there is more than the area form. `two_dim`?